src/a11y-dialog.ts

```diff
@@ -159,6 +135,25 @@ export default class A11yDialog {
     )
   }
 
+  /**
+   * Add a delegated event listener for when elememts that open or close
+   * the dialog are clicked, and call `show` or `hide`, respectively
+   */
+  private bindDelegatedClicks = (evt: Event) => {
```

Comment:
Maybe we call it `event`?

Reply:
how about `bindTriggerClicks`? It's closer to the existing convention of `bind[eventName`]

Reply:
Actually it’s not even binding anything. Maybe `handleDialogVisibility` or something? 😅

Reply:
I would argue that it is binding, insomuch as it is binding an event listener to a DOM node, just as the other functions that begin with `bind` do.

If we want to call this `handle…`, then for that matter it could be `handleTriggerClicks`, and all the other `bind` functions could be `handle` too.

Reply:
Mm, that makes sense. I’ve never thought deeply about the semantics of that naming pattern since I use `handle` to name event listeners in all my own code. I like `handleTriggerClicks` because it indicates that we’re responding to a click event, but I’m happy to call this thing whatever you prefer.

Reply:
Sounds good to me. :)

Reply:
Also just noticed that my very first comment was about naming the parameter `event` instead of `evt`. ;)

Reply:
Oh. 💀 I’m so sorry, i completely misunderstood your original comment. I’m going to go outside and touch grass now.

Reply:
Hahaha, it’s okay! 💚 I’m glad we had a conversation about naming anyway, it’s good. :D